# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/RtpSourceObserver.h
# Commit: d838dc7d2bb8
# Full Hash: d838dc7d2bb89ab5c9c2e8e93895db1c9310560f
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 4
# Description:
#   Bug 1654112 - Update include paths for moved upstream code. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D130059
# ==============================================================================

diff -r 587c40771588 -r d838dc7d2bb8 dom/media/webrtc/libwebrtcglue/RtpSourceObserver.h
--- a/dom/media/webrtc/libwebrtcglue/RtpSourceObserver.h	Fri Oct 23 13:49:51 2020 -0400
+++ b/dom/media/webrtc/libwebrtcglue/RtpSourceObserver.h	Fri Oct 23 14:02:54 2020 -0400
@@ -12,9 +12,11 @@
 
 #include "nsISupportsImpl.h"
 #include "mozilla/dom/RTCRtpSourcesBinding.h"
-#include "webrtc/common_types.h"
 #include "jsapi/RTCStatsReport.h"
 
+#include "api/rtp_headers.h"
+#include "modules/include/module_common_types.h"
+
 // Unit Test class
 namespace test {
 class RtpSourcesTest;